kairos-deposit-contract/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[workspace]
+resolver = "2"
+
+members = [
+  "malicious-reader",
+  "malicious-session",
+  "withdrawal-session",
+  "deposit-session",
```

Comment:
What is the use of those session codes?

Reply:
"Malicious": are used for testing malicious behavior when some unauthorized tries to withdraw from the contract purse

"Deposit-session and Withdrawal-session": Session code to deposit and withdraw from the contract.

Reply:
Since 2.0 is not done yet we have no option to spend funds from the caller purse without using session code.

Reply:
When session code gets deprecated we just replace the session code deploy with a contract call, not a big deal.

Reply:
Ultimately it will all depend on future build targets. For now we have the build target that is locked in the kairos-* branches.